release_notes/372-release-notes.adoc

```diff
@@ -8,4 +8,262 @@ toc::[]
 
 *Release date:* 26 September 2022 +
 
-For the latest release notes for {product-title}, see the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/3.72/html/release_notes/index[{product-title-short} 3.72 Release notes].
+{product-title} ({product-title-short}) is an enterprise-ready, Kubernetes-native container security solution that protects your vital applications across build, deploy, and runtime. It deploys in your infrastructure and integrates with your DevOps tooling and workflows to deliver better security and compliance and to enable DevOps and InfoSec teams to operationalize security.
+
+[id="about-this-release-372"]
+== About this release
+
+{product-title-short} 3.72 includes:
+
+* Automatic removal of inactive clusters
```

Comment:
`inactive` seems broad. We do not manage the removal of clusters that were added but not installed. We only manage removal of those clusters that were installed and did a handshake with Central at least once.
cc: @charmik-redhat